bin/document_slug_updater

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+require "./app/lib/specialist_publisher.rb"
```

Comment:
I think this can just be `require "specialist_publisher"` – the path hoop-jumping is only necessary in the Rails console